### PR TITLE
Condense database character classes

### PR DESCRIPTION
**Pull request by knsjerome**

### CODE REVIEW

**Review by drowsell — APPROVED** [review]
Looks good. Quick work!

File: server/sql/init_db.sql [hunk 1]
```diff
@@ -73,6 +73,7 @@ CREATE TABLE Character (
     ability_points      int NOT NULL CHECK (ability_points >= 0 AND ability_points <= 180), -- Unused ability points
     xp_points           int NOT NULL CHECK (xp_points >= 0 AND xp_points <= 355000),
     class               e_class NOT NULL,
+    class_attribute     text CHECK (length(class_attribute) > 0) NOT NULL,
```
Comment (ksmolko) [inline server/sql/init_db.sql:76]: I was thinking we could combine all the enums below into one, and then make this attribute of type `e_class_attribute` or something like that. That would remove the need for length checking.

Comment (ksmolko) [reply server/sql/init_db.sql:76]: Also, if you were to do that, be sure to add "None" as an option as well (for players to select "None" as their class).

Comment (ksmolko) [reply server/sql/init_db.sql:76]: Otherwise, it would make sense to just remove all the enums below as they are unused.

**Review by ksmolko — CHANGES_REQUESTED** [review]

**Comment by ksmolko** [comment]
Lol looks like I was too late. Perhaps we should follow-up on this?

**Comment by knsjerome** [comment]
> 
> 
> Lol looks like I was too late. Perhaps we should follow-up on this?

Yeah it looks your comments went through about 7-8 minutes after the merge.

The Go code currently validates the value going into the `class_attribute` field, so we should be good to remove the enums from that database or combine them.

**Pull request merged**